src/System.IO.Ports/tests/SerialPort/BreakState.cs

```diff
@@ -12,7 +12,7 @@ namespace System.IO.Ports.Tests
     public class BreakState_Property : PortsTest
     {
         //The maximum time we will wait for the pin changed event to get firered for the break state
-        static readonly int MAX_WAIT_FOR_BREAK = 800;
+        private static readonly int s_MAX_WAIT_FOR_BREAK = 800;
```

Comment:
This can be "private const int". Same for all the other constants that have gotten a `s_` prefix from the formatter. When you rerun the formatter it should remove that prefix.